docs/index.md

```diff
@@ -1376,69 +1376,91 @@ Mocha reporters adjust to the terminal window, and always disable ANSI-escape co
 
 ### Spec
 
-This is the default reporter. The "spec" reporter outputs a hierarchical view nested just as the test cases are.
+**Enable Names:** `Spec`, `spec`
```

Comment:
I'm unsure about "Enable Names", is this an official expression or korean english (I'm not english native neither)? [Here](https://github.com/mochajs/mocha/blob/master/lib/reporters/index.js#L3) it's called `alias`.
I would also prefer a format with less bold letters.